apps/mull-ui/src/app/app.tsx

```diff
@@ -118,6 +119,11 @@ export const App = () => {
               <Redirect to={ROUTES.ANNOUNCEMENTS.url} />
             </PrivateRoute>
           </PrivateRoute>
+          <PrivateRoute path={ROUTES.PROFILE.profile}>
+            <div className="page-container">
+              <PrivateRoute path={ROUTES.PROFILE.editProfile} component={EditProfilePage} />
+            </div>
```

Comment:
You should bring this div into the EditProfilePage component

Reply:
I think something similar should be done. The page-container should be moved to the edit profile component, and a switch should be put here to route between the display and edit routes